.github/workflows/spelling2.yml

```diff
@@ -12,21 +12,83 @@ on:
     tags-ignore:
       - "**"
     types: ['opened', 'reopened', 'synchronize']
+  issue_comment:
+    types: [created]
 
 jobs:
   spelling:
     name: Spell checking
+    permissions:
+      contents: read
+      pull-requests: read
+      actions: read
```

Comment:
This is the read restriction I mentioned in the PR.

Reply:
`actions: read` does not provide read access to secrets?

Reply:
Correct, technically it provides the items listed in this section:
https://docs.github.com/en/rest/overview/permissions-required-for-github-apps#permission-on-actions

(Of course, if a workflow is authorized and it explicitly has `${{ secrets.TOKEN }}` it will get that token when it runs.)

Secrets are guarded by a reasonably well named permission:
https://docs.github.com/en/rest/overview/permissions-required-for-github-apps#permission-on-secrets